app/controllers/events_controller.rb

```diff
@@ -1,14 +1,15 @@
 # frozen_string_literal: true
 
-# Class...
+# Clase del controlador de Eventos
```

Comment:
In my opinion you shouldnt comment your code when its obvious. I mean you can just read the class name why comment it

Reply:
You're right. Fixed.